app/models/event/base.rb

```diff
@@ -2,7 +2,7 @@
 
 class Event::Base
   def notify_sync!(subscriber)
-    subscriber.post_json(to_json)
+    subscriber.do_request(to_json)
   end
 
   def notify_async!(subscriber)
```

Comment:
I know it was already there, but the `Book.current.name == 'central'` is a bit ugly, could be replaced with `Book.current.central?`